api/api.proto

```diff
@@ -1728,48 +1728,65 @@ service OpenStorageFilesystemTrim {
         body: "*"
       };
     }
+
+  rpc AutoFSTrimPush(SdkAutoFSTrimPushRequest)
+      returns (SdkAutoFSTrimPushResponse) {
+    option (google.api.http) = {
+      get : "/v1/filesystem-trim/auto-fstrim-push"
```

Comment:
Shouldn't this be put and not get?

Reply:
Not sure what this API is supposed to do, but if the API is idempotent then PUT otherwise lets use POST.

Reply:
changed to post